### PR TITLE
[tests] update test_git.py to use test base class

### PR DESCRIPTION
**Pull request by aswanipranjal**
This PR updates the `TestGit` class in `tests/test_git.py` file to use the `TestBaseElasticSearch ` class.

### CODE REVIEW

**Comment by aswanipranjal** [comment]
@jgbarah @valeriocos please review?

File: tests/test_git.py [hunk 1]
```diff
@@ -114,11 +113,4 @@ def test_overview_timeseries_with_df(self):
...
         assert_array_equal(authors_test['value'], authors['value'])
-
-    @classmethod
-    def tearDownClass(cls):
-        """
-        Destroy the infrastructure created for unittests
-        """
-
-        teardown_data_source(cls.git[0], cls.git[1])
+        assert_array_equal(authors_test['value'], authors['value'])
```
Comment (sduenas) [inline tests/test_git.py]: Why do you need to check this twice? The previous line checks the same.

Comment (aswanipranjal) [reply tests/test_git.py]: Thanks for pointing this out!

**Comment by valeriocos** [comment]
LGTM

**Comment by aswanipranjal** [comment]
@valeriocos can we merge this too, please?

**Pull request merged**